source/includes/api/_organizations.md

```diff
@@ -1,180 +1,183 @@
 # Organizations
 
-## The organization object
-
-```json
-{
-  "id": "eb4dfe83-f1fd-46dd-a69d-b7cf7b566319",
-  "data": {
-    "name": "Organization Test",
-    "token": "FZsvUJCzCS",
-    "parentId": "28a699ee-342d-4bae-af37-c07924c0e23e",
-    "description": "Organization Test",
-    "discoverable": false,
-    "editOldTurfs": true,
-    "viewUnassignedData": true,
-    "requireSignUpConfirmation": true
-  },
-  "meta": {
-    "etag": "27d-wlAWROPGf7xiGd2BZNuoSaeMLlE",
-    "created": "2018-09-18T21:10:45.619Z",
-    "modified": "2018-09-18T21:10:45.619Z",
-    "resource": "organizations",
-    "createdBy": "13b6fc16-b84f-42a0-ad84-1135e12e0851",
-    "isDeleted": false,
-    "modifiedBy": "13b6fc16-b84f-42a0-ad84-1135e12e0851"
-  },
-  "customerId": "f51b86dd-aaba-485e-b14d-65cbea28b8e4",
-  "securityGroupId": "8e9363ba-215c-40cb-822f-639e6b957472"
-}
-```
+## The Organization object
 
 Attribute | Required? | Description
 --------- | --------- | -----------
-id | true | Unique identifier for the object.
-customerId | true | Customer this resource belongs to.
-securityGroupId | true | Security group of this resource.
-data | true | Organization data.
+id | true | Identifier of this record
+securityGroupId | true | Identifier of the associated securitygroup
+customerId | true | Identifier of the associated customer
+data | true | Organization data
 
 ### data
 
 Attribute | Required? | Description
 --------- | --------- | -----------
-name | true | Name of the organization.
-description | false | Descriptions of the role.
-permissions | false | Array of permissions allowed to that role.
-token | false | Token used to join an organization.
-parentId | false | Id of the parent Organization.
-discoverable | false | Defines if the organization is public. Default `false`.
-editOldTurfs | false | Enables editting old turfs. Default `true`.
-viewUnassignedData | false | Enables canvassers to view unassigned data. Default `true`.
-requireSignUpConfirmation | false | Joining organization by link, if `false`, doesn't need to be approved by a manager. Default `true`.
+name | true | Name of the organization
+description | false | Description text of the organization
+discoverable | true | Flag indicating whether the organization is public. Default `false`
+editOldTurfs | false | Flag indicating whether the old turfs belonging to this organization are editable. Default `true`
+viewUnassignedData | false | Enables canvassers to view unassigned data. Default true
+parentId | false | Id of the `parent` Organization
+token | false | Token used to join an organization
+requireSignUpConfirmation | false | Joining organization by link, if `false`, doesn't need to be approved by a manager. Default `true`
 
 ### meta
 
-[See documentation](#metadata-object).
```

Comment:
nothing was changed in organization, so don't commit these changes

Reply:
we removed permissions from schema in endgame branch